test/PlaidClientTest.js

```diff
@@ -408,7 +408,7 @@ describe('plaid.Client', () => {
           getAllTransactionsWithRetries(accessToken, now, now, 5,
           (err, transactions) => {
             expect(err).to.be(null);
-            expect(transactions).to.be.an(Array);
+            expect(transactions.transactions).to.be.an(Array);
```

Comment:
can we also check that `transactions.accounts` `transactions.item` and `transactions.total_transactions` are not null/undefined?